riva/clients/nmt/BUILD

```diff
@@ -91,12 +91,12 @@ cc_library(
 )
 
 cc_binary(
-    name = "riva_streaming_speech_translate_client",
-    srcs = ["riva_streaming_speech_translate_client.cc"],
+    name = "riva_streaming_s2t_client",
```

Comment:
do we have offline speech to text client ?
can we all clients as riva_nmt_* ?
example riva_nmt_streaming_s2t_client ?

Reply:
we don't have offline client for s2s and s2t, currently only streaming is there.

Yes, I thought about that. But since s2s/s2t also use asr or tts in addition to nmt, and also customers are visualizing as separate service like ASR/NMT/TTS, think it is better to not prefix with nmt. NMT is seen as just text to text.

Reply:
updated names

Reply:
looks fine